site/content/en/reference/live/apply/_index.md

```diff
@@ -22,12 +22,13 @@ it consistent with other kpt commands.
 
 #### `kubectl apply` vs `kpt live apply`
 
-|                     | `kubectl apply`            | `kpt live apply`          |
-|---------------------|----------------------------|---------------------------|
-|Usage                | kubectl apply -f /dir      | kpt live apply /dir       |
-|Applied resource set | Not tracked                | Tracked                   |
-|Prune                | Imperative and error prone | Declarative and reliable  |
-|Status               | Not supported              | Supported                 |
+|                     | `kubectl apply`                          | `kpt live apply`                        |
+|---------------------|------------------------------------------|-----------------------------------------|
+|Usage                | kubectl apply -f /dir                    | kpt live apply /dir                     |
+|                     | kubectl apply --server-side -f /dir      | kpt live apply --server-side /dir       |
```

Comment:
I don't think we need to cover this here. This is to highlight the pruning and status functionality.